yb-voyager/cmd/common.go

```diff
@@ -1151,6 +1153,8 @@ type AssessMigrationPayload struct {
 	SourceSizeDetails              SourceDBSizeDetails
 	TargetRecommendations          TargetSizingRecommendations
 	ConversionIssues               []utils.AnalyzeSchemaIssue
+	Sizing                         *migassessment.SizingAssessmentReport
```

Comment:
I understand it's this way even in the main assessmentReport struct, but why pointers for both of these fields?

Reply:
Not sure, maybe to save copies in memory. But i guess we didn't follow anything strictly for the fields.

Reply:
added a TODO to revisit it later.